src/ui/backend/canvas/animate.js

```diff
@@ -35,25 +34,24 @@ exports = function (subject, groupId) {
 		import device;
 	}
 
-	if (device.useDOM && subject instanceof View && !groupId) {
+	if (device.useDOM && subject instanceof View && !groupID) {
 		return subject.getAnimation();
 	}
 
-	var groupId = groupId || 0,
-		group = groups[groupId] || (groups[groupId] = new Group()),
-		animID = subject.__anim_id || (subject.__anim_id = '__anim_' + (++anim_uid)),
-		anim = group.get(animID);
-
+	var groupID = groupID || 0;
+	var animID = subject.__anim_id || (subject.__anim_id = '__anim_' + (++uid));
+	var anims = subject.__anims || (subject.__anims = {});
```

Comment:
Should at least add a note in the comments: Subjects that are not GCed could potentially have many items in `__anims` if there are many unique group IDs